src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -0,0 +1,264 @@
+                                   QuantizedPtNodeType quantization_type) {
+    return quantize(context,
+                    input.get_node_shared_ptr(),
+                    scale.get_node_shared_ptr(),
+                    zero_point.get_node_shared_ptr(),
+                    nullptr,
+                    dtype,
+                    quantization_type);
+}
+
+ov::Output<ov::Node> quantize(const NodeContext& context,
+                                   ov::Output<ov::Node> input,
+                                   ov::Output<ov::Node> scale,
+                                   ov::Output<ov::Node> zero_point,
+                                   ov::Output<ov::Node> axis,
+                                   ov::element::Type dtype,
+                                   QuantizedPtNodeType quantization_type) {
+    return quantize(context,
+                    input.get_node_shared_ptr(),
+                    scale.get_node_shared_ptr(),
+                    zero_point.get_node_shared_ptr(),
+                    axis.get_node_shared_ptr(),
+                    dtype,
+                    quantization_type);
+}
+
+ov::Output<ov::Node> quantize(const NodeContext& context,
+                                   ov::Output<ov::Node> input,
+                                   ov::Output<ov::Node> quantized_node) {
+    std::shared_ptr<QuantizedPtNode> quantized_pt_node;
+    if ((quantized_pt_node = cast_quantized_fw_node(quantized_node.get_node_shared_ptr()))) {
+        return quantize(context,
+                        input.get_node_shared_ptr(),
+                        quantized_pt_node->get_scale(),
+                        quantized_pt_node->get_zero_point(),
+                        quantized_pt_node->get_axis(),
+                        quantized_pt_node->get_input_element_type(0),
+                        quantized_pt_node->get_type());
+    }
+    FRONT_END_OP_CONVERSION_CHECK(false, "Failed to convert a node to QuantizedPtNode");
+}
+
+ov::Output<ov::Node> quantize(const NodeContext& context,
+                                   ov::Output<ov::Node> input,
+                                   ov::Output<ov::Node> scale,
+                                   ov::Output<ov::Node> zero_point,
+                                   ov::Output<ov::Node> quantized_node) {
+    std::shared_ptr<QuantizedPtNode> quantized_pt_node;
+    if ((quantized_pt_node = cast_quantized_fw_node(quantized_node.get_node_shared_ptr()))) {
+        return quantize(context,
+                        input.get_node_shared_ptr(),
+                        scale.get_node_shared_ptr(),
+                        zero_point.get_node_shared_ptr(),
+                        quantized_pt_node->get_axis(),
+                        quantized_pt_node->get_input_element_type(0),
+                        quantized_pt_node->get_type());
+    }
+    FRONT_END_OP_CONVERSION_CHECK(false, "Failed to convert a node to QuantizedPtNode");
+}
```

Comment:
These functions are not used in this PR, please add them in the PR where you need them.

Reply:
Moved to the quantized ops PR!